docs/staff/index.md

```diff
@@ -0,0 +1,48 @@
+
+<StaffBoard uid="30973609" userName="Amrzs0971" role="程序"></StaffBoard>
+<StaffBoard uid="33887163" userName="kitakore" role="裁判，直播，规则"></StaffBoard>
+
+
+
+|头像|ID|职务|
+|:-:|:-|:-|
+|<img src="https://a.ppy.sh/33887163?1720800005.jpeg" width="100"/>|[kitakore](https://osu.ppy.sh/users/33887163)|裁判，直播，规则| 
+|<img src="https://a.ppy.sh/30595368?1720799710.jpeg" width="100"/>|[Paraliesa](https://osu.ppy.sh/users/30595368)|表格|
+|<img src="https://a.ppy.sh/30895664?1715264556.jpeg" width="100"/>|[Dr_Shindo](https://osu.ppy.sh/users/30895664)|裁判，解说，直播|
+|<img src="https://a.ppy.sh/34282059?1722777886.jpeg" width="100"/>|[5atori Komeiji](https://osu.ppy.sh/users/34282059)|美工，解说|
+|<img src="https://a.ppy.sh/33295812?1722717131.jpeg" width="100"/>|[Koishi Komeiji](https://osu.ppy.sh/users/33295812)|美工，裁判|
+|<img src="https://a.ppy.sh/29235972?1722541681.jpeg" width="100"/>|[FroZZZZZen](https://osu.ppy.sh/users/29235972)|裁判|
+|<img src="https://a.ppy.sh/33418980?1720533238.jpeg" width="100"/>|[Mimori](https://osu.ppy.sh/users/33418980)|选图|
+|<img src="https://a.ppy.sh/13217389?1717553951.jpeg" width="100"/>|[My Angel Kuroe](https://osu.ppy.sh/users/13217389)|选图，规则|
+|<img src="https://a.ppy.sh/15349720?1720607917.jpeg" width="100"/>|[Naughty_chas](https://osu.ppy.sh/users/15349720)|裁判，直播，程序|
+|<img src="https://a.ppy.sh/15175276?1709356004.jpeg" width="100"/>|[_eilphy](https://osu.ppy.sh/users/15175276)|测试，裁判，解说|
+|<img src="https://a.ppy.sh/12438405?1715487727.jpeg" width="100"/>|[dynamix player](https://osu.ppy.sh/users/12438405)|选图，测试，裁判|
+|<img src="https://a.ppy.sh/14817468?1705334951.jpeg" width="100"/>|[stllok](https://osu.ppy.sh/users/14817468)|选图，测试，裁判，程序|
+|<img src="https://a.ppy.sh/27921376?1723126636.jpeg" width="100"/>|[mayoi shigure](https://osu.ppy.sh/users/27921376)|选图|
+|<img src="https://a.ppy.sh/13888511?1719592414.jpeg" width="100"/>|[ShandenLunaire](https://osu.ppy.sh/users/13888511)|选图，测试，裁判，解说，直播|
+|<img src="https://a.ppy.sh/5237349?1712837187.jpeg" width="100"/>|[sourflour](https://osu.ppy.sh/users/5237349)|规则，表格|
+|<img src="https://a.ppy.sh/16901335?1722324503.jpeg" width="100"/>|[hinanawi momoko](https://osu.ppy.sh/users/16901335)|选图|
+|<img src="https://a.ppy.sh/13795588?1712406732.jpeg" width="100"/>|[lelxdme](https://osu.ppy.sh/users/13795588)|选图，测试，裁判，解说|
+|<img src="https://a.ppy.sh/28412976?1722778278.jpeg" width="100"/>|[L1ttle_sky](https://osu.ppy.sh/users/28412976)|美工|
+|<img src="https://a.ppy.sh/30973609?1707497304.jpeg" width="100"/>|[Amrzs0971](https://osu.ppy.sh/users/30973609)|程序|
+|<img src="https://a.ppy.sh/7210268?1715498207.jpeg" width="100"/>|[Tomoe](https://osu.ppy.sh/users/7210268)|选图，测试|
+|<img src="https://a.ppy.sh/14721101?1717747082.jpeg" width="100"/>|[cdwcgt](https://osu.ppy.sh/users/14721101)|程序|
+|<img src="https://a.ppy.sh/30329943?1723016654.jpeg" width="100"/>|[Nosenso](https://osu.ppy.sh/users/30329943)|解说|
+
+
+
+
+
+<img src="https://a.ppy.sh/33887163?1720800005.jpeg" width="70"/> 
+
```

Comment:
These lines seem duplicate.

Should we remove them?

Reply:
OK, I'll do that myself later...

Reply:
I suggest to leave these until we have fully transfer these information onto the new staffBoard design.